packages/cli/src/utils/build.js

```diff
@@ -497,6 +499,7 @@ const buildAndOrUpload = async (
   let app;
   if (upload) {
     app = await getWritableApp();
+    checkMissingAppInfo(app);
```

Comment:
We probably don't want to do this in `build`. `zapier build` only generates the zip file and doesn't save the app. We should allow the developer to build the zip even if there's missing app info.